### PR TITLE
alternate Dimension

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/webapp/app/services/visualizations/clusteredhorizontalbar-chart.service.js [hunk 1]
```diff
@@ -49,6 +49,7 @@
                     result['isFilterGrid'] = VisualizationUtils.getPropertyValue(record.properties, 'Show Filter Grid');
                     result['showSorting'] = VisualizationUtils.getPropertyValue(record.properties, 'Allow Sorting');
                     result['displayName'] = VisualizationUtils.getFieldPropertyValue(dimensions[0], 'Display name') || result['dimension'][0];
+                    result['alternateDimension'] = VisualizationUtils.getPropertyValue(record.properties, 'Alternative Dimensions');
```
Comment (johibkhan2) [inline src/main/webapp/app/services/visualizations/clusteredhorizontalbar-chart.service.js:52]: why we are keeping visualization related files inside flair-bi...we have them already in flair-visualisation project

Comment (khushbu-maurya) [reply src/main/webapp/app/services/visualizations/clusteredhorizontalbar-chart.service.js:52]: it's not visualization related files 
its visualization service file that calls flair-visualisation

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**